src/Host/Broker/Impl/About/AboutController.cs

```diff
@@ -14,7 +14,7 @@
 
 namespace Microsoft.R.Host.Broker.About {
     [Authorize(Policy = Policies.RUser)]
-    [Route("/about")]
```

Comment:
Why this change? We've used lowercase for HTTP paths everywhere else, and it is also the standard REST convention.